cmd/sriov/main.go

```diff
@@ -221,6 +221,10 @@ func cmdDel(args *skel.CmdArgs) error {
 
 	sm := sriov.NewSriovManager()
 
+	if err := sm.ResetVFConfig(netConf); err != nil {
```

Comment:
can you add some info in commit message a bout this change as well as you only cover the issue, and the change with retry.